documentation/operators/src/sandbox.md

```diff
@@ -10,21 +12,25 @@ This page is for Nym node operators. If you want to run NymVPN CLI over Sandbox
 
 > Any syntax in `<>` brackets is a user's unique variable. Exchange with a corresponding name without the `<>` brackets.
 
-To run Nym binaries in Sandbox Testnet you need to get the `.env` configuration file and point your binary to it. Follow the steps below:
+To run Nym binaries in Sandbox Testnet you need to get `sandbox.env` configuration file and point your binary to it. Follow the steps below:
```

Comment:
To run Nym binaries in the sandbox testnet, you need to get the sandbox.env configuration file and point your binary to it `the inclusion of the comma after testnet`